nodejs/express/books/Dockerfile

```diff
@@ -1,25 +1,17 @@
-FROM registry.access.redhat.com/ubi8/nodejs-10
+FROM registry.access.redhat.com/ubi8/nodejs-14
 WORKDIR /app
 COPY . .
 
-RUN id
 USER 0
-RUN yum repolist
-# utils
-RUN yum -y install vim
+# Install EPEL https://fedoraproject.org/wiki/EPEL for autossh
+RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+RUN yum -y install unixODBC-devel gcc autossh
 # odbc driver
-RUN yum -y install unixODBC-devel gcc
-RUN yum -y install python3 && ln -sf python3 /usr/bin/python
-RUN curl -vvv -k 'https://w3.rchland.ibm.com/projects/CA400/Win32/process/express/drivers/pool/v7r2m0/v0001/IBMiAccess_v1r1_LinuxAP_u14.zip' -o ap.zip
-RUN unzip -j ap.zip \*x86_64.rpm
+RUN unzip -j IBMiAccess\*.zip \*x86_64.rpm
 RUN yum -y  install  ./ibm-iaccess-*.rpm
```

Comment:
need to blast ibm-iaccess-32bit*

Reply:
You wouldn't _need_ to, but there's no reason to pull in all the 32-bit libs that wouldn't be used.

Reply:
They fail to compile because of (IIRC) competing compiler requirements

Reply:
Updated to remove 32 bit rpms